lms/static/scripts/frontend_apps/components/StudentSelector.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  IconButton,
+  InputGroup,
+  Select,
+} from '@hypothesis/frontend-shared/lib/next';
+import classnames from 'classnames';
+
+type Student = {
+  displayName: string;
+};
+
+export type StudentSelectorProps = {
+  /** Callback invoked when the selected student changes */
+  onSelectStudent: (index: number) => void;
+  /** Index of selected student in `students` or -1 if no student is selected */
+  selectedStudentIndex: number;
+  /** Ordered list of students to display in the drop-down */
+  students: Student[];
+};
+
+/**
+ * A student navigation tool which shows a student selection list and a previous and next button.
```

Comment:
I know this was copied from the previous code, but I think the wording here could be refined. Maybe something like "Control which allows selecting a student from a list."